vpr/src/route/route_budgets.cpp

```diff
@@ -100,6 +109,11 @@ void route_budgets::load_initial_budgets() {
             //before all iterations, delay max budget is equal to the lower bound
             delay_max_budget[net_id][ipin] = delay_lower_bound[net_id][ipin];
             delay_min_budget[net_id][ipin] = delay_lower_bound[net_id][ipin];
+            should_reroute_for_hold[net_id] = false;
+
+            short_path_crit[net_id][ipin] = 1;
+            total_path_delays_hold[net_id][ipin] = -2;
+            total_path_delays_setup[net_id][ipin] = -2;
```

Comment:
These numbers need some clarifications. I think they represent INVALID something, but you should probably define macros for them.

Reply:
Yes, should definitely use named constants instead of -2.

Reply:
Good catch, they are supposed to mean invalid/unset. I can change that.